src/App.js

```diff
@@ -29,14 +29,24 @@ class App extends Component {
     }
   }
 
+  inputEvent = (e) => {
```

Comment:
so right now, pull request means you're trying to merge that branch into master
You've done your changes, and you feel like it's bug proof and ready to be applied to the rest of the app so you make pull request, where your co-programmers check if its good and you can than merge it.